translations/sv/learn.json

```
@@ -0,0 +1,31 @@
+{
+  "diving_into_fediverse": "Dyka ner i <mark>fediversumet</mark>",
+  "what_is_fediverse": "<p><a href='https://en.wikipedia.org/wiki/Fediverse'>Fediversumet</a> är <mark>enormt</mark>. Det består av tusentals gemenskaper med miljontals människor som du och jag, som vill <mark>ansluta och dela</mark>.</p><p>Här är några resurser som hjälper dig att få en bättre uppfattning om hur saker fungerar, hur du hittar en gemenskap som passar dig och utforskar verktyg du kan använda för att göra din upplevelse ännu bättre.</p>",
```

```suggestion
  "what_is_fediverse": "<p><a href='https://sv.wikipedia.org/wiki/Fediversum'>Fediversumet</a> är <mark>enormt</mark>. Det består av tusentals gemenskaper med miljontals människor som du och jag, som vill <mark>umgås och dela med sig</mark>.</p><p>Här är några resurser som hjälper dig att få en bättre uppfattning om hur saker fungerar, hur du hittar en gemenskap som passar dig och utforskar verktyg du kan använda för att göra din upplevelse ännu bättre.</p>",
```

* Link to Swedish wikipedia
* "ansluta" have a technical tone, it is not somthing you do between humans. Reworded to something like "socialize and share"